src/auth/basic/SSPI/basic_sspi_auth.cc

```diff
@@ -149,7 +149,7 @@ main(int argc, char **argv)
         /* Clear any current settings */
         username[0] = '\0';
         password[0] = '\0';
-        sscanf(wstr, "%s %s", username, password);  /* Extract parameters */
+        sscanf(wstr, "%255s %255s", username, password);  /* Extract parameters */
```

Comment:
This does not work in my quick-and-dirty [tests](https://godbolt.org/z/jjMdK3oor): If username input field is too long, then `password` gets the extra characters from the username input field rather than skipping username trailer and reading the password field.